config/samples/secret-agent_v1alpha1_secretagentconfiguration.yaml

```diff
@@ -44,6 +85,19 @@ spec:
     - name: authorized_keys
       type: publicKeySSH
       privateKeyPath: ["amster", "id_rsa"]
+  - name: truststore
+    keys:
+    - name: cacerts
+      type: truststore
+      keyStorePath: ["truststore", "cacerts-keypin"]
+      storePassPath: ["truststore", "cacerts-keypin"]
+      keystoreAliases:
+        - alias: ca-cert
+          type: caCopy
+          caPath: ["platform-ca-public", "public-key"]
+    - name: cacerts-keypin
```

Comment:
@wstrange do you know where AM/IDM look for the pin for the truststore?

Reply:
"storePassword": "&{openidm.truststore.password|changeit}",

is in the idm config

Reply:
Im guessing the truststore pin is hardcoded to changeit